src/main/webapp/WEB-INF/views/drivers/add.jsp

```diff
@@ -5,28 +5,37 @@
 </style>
 <html>
 <head>
-    <title>All drivers</title>
+    <title>Taxi Service</title>
 </head>
 <body>
-<form method="post" id="driver" action="${pageContext.request.contextPath}/drivers/add"></form>
-<h1 class="table_dark">Add driver:</h1>
-<table border="1" class="table_dark">
-    <tr>
-        <th>Name</th>
-        <th>License number</th>
-        <th>Add</th>
-    </tr>
-    <tr>
-        <td>
-            <input type="text" name="name" form="driver" required>
-        </td>
-        <td>
-            <input type="text" name="license_number" form="driver" required>
-        </td>
-        <td>
-            <input type="submit" name="add" form="driver">
-        </td>
-    </tr>
-</table>
+<%@include file='/WEB-INF/views/header.jsp'%>
+    <form method="post" id="driver" action="${pageContext.request.contextPath}/drivers/add"></form>
+        <h1 class="table_dark">Add driver:</h1>
+        <table border="1" class="table_dark">
+            <tr>
+                <th>Name</th>
+                <th>License number</th>
+                <th>Login</th>
+                <th>Password</th>
```

Comment:
password output is bad practice, let's remove it

Reply:
yes, but it is not up to me, the condition says that our driver will be as user, so how can we create a new driver without this data?